load-test-framework/src/main/java/com/google/pubsub/clients/gcloud/CPSPublisherTask.java

```diff
@@ -52,10 +54,9 @@ private CPSPublisherTask(String project, String topic, int messageSize, int batc
   }
 
   public static void main(String[] args) throws Exception {
-    LoadTestRunner.run(request ->
+    LoadTestRunner.run((Function<StartRequest, Task>) request ->
```

Comment:
This isn't necessary for the compiler, but I added it 1. to silence an Eclipse warning, 2. I think it makes it more readable when you know what request is without having to go to LoadTestRunner to check.